region/tourian/main/Metroid Room 3.json

```diff
@@ -359,20 +359,71 @@
       },
       "requires": [
         "canMetroidAvoid",
+        "canTrickyDodgeEnemies",
         "h_artificialMorphSpringBall",
-        {"ammo": {"type": "PowerBomb", "count": 4}}
+        {"ammo": {"type": "PowerBomb", "count": 3}},
+        {"or": [
+          "canTrickyGMode",
+          {"ammo": {"type": "PowerBomb", "count": 1}}
+        ]}
       ],
       "setsFlags": ["f_KilledMetroidRoom3"],
       "flashSuitChecked": true,
       "note": [
         "Kill the Metroids while artificially morphed with just Spring Ball to avoid them.",
-        "A Metroid (or a Rinka) must be completely on screen while a Power Bomb explodes in order for it to lure the other Metroids.",
+        "A Rinka must be completely on screen while a Power Bomb explodes in order for it to lure the Metroids.",
         "One strategy for this location is to wait for the Rinkas to fire twice, then place a Power Bomb in the hallway to the right and return to lure one.",
         "Jump over it twice and at the top of the second jump place a second Power Bomb; this one will explode when the other two Metroids arrive.",
         "Jumping high to the right can also help get them stuck in the top right corner while placing a Power Bomb.",
+        "Note that it is necessary to exit G-Mode before exiting the room in order for the Metroids to remain killed.",
+        "With only 3 Power Bombs, cluster the Metroids before laying Power Bombs."
+      ]
+    },
+    {
+      "link": [1, 1],
+      "name": "G-Mode Morph Power Bomb Metroid Kill, No Movement Items",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "canMetroidAvoid",
+        "canTrickyGMode",
```

Comment:
The instructions work, so its not that insane to do.  Up to you if you want to keep the tech here.